Update functions-monitoring.md

tsahi opened the pull request:
add a path to where the log file is located in the storage account. see Issue #5831

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ostrich75 commented:
#sign-off

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@ostrich75 - See mail about working in the private repo, please

tsahi committed:
Update functions-monitoring.md

add a path to where the log file is located in the storage account. see Issue #5831
